swbundle/p2g_compare_geotiff.sh

```diff
@@ -59,34 +59,13 @@ BAD_COUNT=0
 for VFILE in $VERIFY_BASE/*.tif; do
     WFILE=$WORK_DIR/`basename $VFILE`
     if [ ! -f $WFILE ]; then
-        oops "ERROR: Could not find output file $WFILE"
+        oops "Could not find output file $WFILE"
         BAD_COUNT=$(($BAD_count + 1))
         continue
     fi
-    echo "Comparing $WFILE to known valid file"
-    python<<EOF
-from osgeo import gdal
-import numpy
-import sys
-
-work_gtiff  = gdal.Open("$WFILE", gdal.GA_ReadOnly)
-valid_gtiff = gdal.Open("$VFILE", gdal.GA_ReadOnly)
-
-work_data = work_gtiff.GetRasterBand(1).ReadAsArray()
-valid_data = valid_gtiff.GetRasterBand(1).ReadAsArray()
-
-if work_data.shape != valid_data.shape:
-    print("ERROR: Data shape for '$WFILE' is not the same as the valid '$VFILE': {0}, {1}".format(work_data.shape, valid_data.shape))
-    sys.exit(1)
-
-total_pixels = work_data.shape[0] * work_data.shape[1]
-equal_pixels = len(numpy.nonzero( work_data == valid_data )[0])
-if equal_pixels != total_pixels:
-    print ("FAIL: %d pixels out of %d pixels are different" % (total_pixels-equal_pixels,total_pixels))
-    sys.exit(2)
-print ("SUCCESS: %d pixels out of %d pixels are different" % (total_pixels-equal_pixels,total_pixels))
-EOF
-    [ $? -eq 0 ] || BAD_COUNT=$(($BAD_COUNT + 1))
+    echo "INFO: Comparing $WFILE to known valid file"
+    python -m polar2grid.compare geotiff "$VFILE" "$WFILE" --atol 0. -vv
```

Comment:
I think we talked about maybe letting users pass additional arguments by passing `"$@"` here, right? Then users could add more `-v` if they wanted or change `--atol` themselves. I think `atol` should probably default to `0` in the python script.

For the argument passing you could look at using the `shift 1` command when the arguments are read above (the `VERIFY_BASE` and `WORK_DIR` part). This would make `$@` be the arguments we care about. Thoughts?